sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -1297,7 +1282,6 @@ def test_RadiusNeighborsRegressor_multioutput_with_uniform_weight():
     X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
 
     for algorithm, weights in product(ALGORITHMS, [None, "uniform"]):
-
```

Comment:
Those lines deletion should be reverted.

Do you know if this was made by `black`?

Reply:
I just reverted the `black` changes, there seem to be different changes by black made based on the version of black.